### PR TITLE
[TASK] Move linting to a Composer script and use it on CI

### PR DESCRIPTION
**Pull request by ziegenberg**
Also lint `config/` in order to catch e.g. language features that are not supported in all the PHP versions we support.

Fixes: #551

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -69,11 +69,13 @@
         "ci:dynamic": [
             "@ci:tests"
         ],
-        "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests",
-        "ci:php:stan": "phpstan --no-progress --configuration=config/phpstan.neon",
+        "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests config",
+        "ci:php:lint": "find src tests config bin -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
```
Comment (oliverklee) [inline composer.json:73]: Please let's also have a line in the `scripts-description` section of the `composer.json`.

**Review by oliverklee — CHANGES_REQUESTED** [review]
Thanks! Looks good! One thing to add to get this to mergeable.

File: composer.json (hunk 1 above)
Comment (ziegenberg) [reply composer.json:73]: Fixed.

**Review by oliverklee — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**